backend/src/packages/users/user.repository.ts

```diff
@@ -16,23 +26,60 @@ class UserRepository implements Repository {
   public async findAll(): Promise<UserEntity[]> {
     const users = await this.userModel.query().execute();
 
-    return users.map((user) => UserEntity.initialize(user));
+    const userPromises = users.map(async (user) => {
```

Comment:
Join in one request

Reply:
Ok, I followed the same logic as [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/46#discussion_r1303162638)

Reply:
Sorry, didn't get what you mean, not to use this:  `const users = await this.userModel.query().execute();` ?

Reply:
.query().select().withGraphJoined('userDetails');